src/app/components/turnos/gestor-agendas/pipes/botonesAgendaGeneral.pipe.ts

```diff
@@ -0,0 +1,96 @@
+import { Auth } from '@andes/auth';
+import { Pipe, PipeTransform } from '@angular/core';
+import { forEach } from 'vis-util';
+import { IAgenda } from '../../../../interfaces/turnos/IAgenda';
+
+@Pipe({
+    name: 'botonesAgendaGeneral'
+})
+export class BotonesAgendaGeneralPipe implements PipeTransform {
+    constructor(private auth: Auth) { }
+    transform(agendas: IAgenda[]): any {
+        const botones = {
+            publicar: false,
+            borrar: false,
+            disponible: false,
+            reanudar: false,
+            imprimir: false,
+            imprimirCarpetas: false,
+            nota: false,
+            pausar: false
+        };
+
+        const puedeImprimir = this.auth.check('turnos:agenda:puedeImprimir:');
+        const puedeNota = this.auth.check('turnos:agenda:puedeNota:');
+        const puedePausar = this.auth.check('turnos:agenda:puedePausar:');
+        const puedeHabilitar = this.auth.check('turnos:agenda:puedeHabilitar:');
+        const puedeBorrar = this.auth.check('turnos:agenda:puedeBorrar:');
+        const puedeReanudar = this.auth.check('turnos:agenda:puedeReanudar:');
```

Comment:
idem anterior
los dos puntos finales están de mas
`this.auth.check('turnos:agenda:puedeEditar');`

Reply:
Corregido